app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -928,6 +928,7 @@ def disable_button(id)
       when "vm_snapshot_delete_all"
         return @record.is_available_now_error_message(:remove_all_snapshots) unless @record.is_available?(:remove_all_snapshots)
       when "vm_snapshot_revert"
+        return N_("Select a snapshot that is not the active one") if @active
```

Comment:
Can this be added here https://github.com/durandom/manageiq/blob/06c8bc3d189fc6277cd4933bb274e02d654badb0/app/models/manageiq/providers/redhat/infra_manager/vm/operations/snapshot.rb#L50-L50
?